Elevating centre of Bertha so that terraformed Berthas are targettable

danfireman opened the pull request:
Currently Redbacks, Scorpions and Fleas can't target Berthas that have been terraformed up on high spikes. This is because the targetting centre of the Bertha is so low that the ground of the platform it's on blocks line of sight.

This pull request elevates the Berthas centre so that Fleas, Redbacks and Scorpions will be able to fire at it, while leaving the collision volumes and selection volumes untouched.

GoogleFrog commented:
Change the aimpos not the midpos.

sprunk commented:
There's a way to keep the aimpos low on un-terraformed buildings (for scythes etc) and only raise it when terraformed. See https://github.com/ZeroK-RTS/Zero-K/blob/0372f4db69bc6a7c2d55f7bf370ab31bba42f645/scripts/turretheavylaser.lua#L50

danfireman commented:
@GoogleFrog : Changed to aimpoint. Also properly centred aimpoint and hitbox. Also added wreck hitbox while in the area.

@sprunk : As discussed on Discord, not too sure how to get that working.

GoogleFrog commented:
I don't think the GG is required since BB is rarely damaged as part of a knife-edge battle. Does this aimpoint work at all build orientations? I recall there being an issue with horizontal offsets for aim and midpoints.

danfireman commented:
Aimpoint dot, collision volumes and where a Redback fires seem consistent and sensible in all your orientations.
